aries-site/src/pages/components/tip.mdx

```diff
@@ -59,3 +74,11 @@ A Tip can be used to reiterate traditional symbols.
 <Example code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tooltip/ToolTipExitExample.js">
   <ToolTipExitExample />
 </Example>
+
+### Tip Sizes
```

Comment:
I feel like using "Size" here is a little misleading because typically when we refer to size we mean a prop `size` that adjust dimensions/font sizing/padding/etc. This is more about content length and less about a `size` prop on the actual component. I wonder if it would be better to label this something more like "Tip Content".
```suggestion
### Tip Content
```

Reply:
Also, is there any guidance around what's the maximum amount of text that should go in a tip like this?

Reply:
yeah good suggestion I can look in the figma but I don't think I saw any

Reply:
Aditi is going to explore the Maximum amount of text and I will add in the guidance once that is there!
Keep track here https://github.com/grommet/hpe-design-system/issues/1793